pages/operations/uninstall.md

```diff
@@ -28,3 +39,26 @@ plural destroy
 {% callout severity="danger" %}
 Only do this if you're absolutely sure you want to bring down all associated resources with this repository.
 {% /callout %}
+
+By default, previously installed applications will still appear in app.plural.sh after running `plural destroy`. To remove all installation history, run `plural repos reset` as documented below.
+
+## Terraform Destroy
+
+To tear down the current cluster but leave installation references as pointers to the helm/terraform, cd into `bootstrap/terraform` and run:
+
+```
+terraform destroy
+```
+
+
+## Start from Zero
+
+To remove all installation references in our API, run:
+
+```
+plural repos reset
+```
+
+This command does not interact with any infrastructure, but removes references to all installations. This will also reset any association with a specific cloud provider.
+
+To tear down a cluster and fully start over, run `plural destroy` and then `plural repos reset`.
```

Comment:
Is it worth having a subheading that says "Fully Start Over" or "Destroy Everything" for these 2 commands together?